src/app/lib/config/toggles/localConfig.js

```diff
@@ -50,4 +50,7 @@ export default {
   variantCookie: {
     enabled: true,
   },
+  webVitalsMonitoring: {
```

Comment:
isn't this the addition of a toggle to the toggles service?

Reply:
The WebVitals component checks if the `webVitalsMonitoring` toggle is true, and only then will it run the Web Vitals code. 

Because there was no default value defined for that toggle in the local environment, this would always be false, and we would have no way of testing it on localhost.

Reply:
removed console log